python/vTestInterface.py

```diff
@@ -511,15 +520,18 @@ def validateClicastCommand(command, mode):
     """
     if mode in ["executeTest", "rebuild"]:
         if command is None or len(command) == 0:
-            print(f"Arg --clicast is required for mode: {mode}")
-            raise UsageError()
+            raise UsageError("--clicast argument is required")
         elif os.path.isfile(command) or (
             sys.platform == "win32" and os.path.isfile(command + ".exe")
         ):
             pass
         else:
-            print(f"Invalid value for --clicast: {command}")
-            raise UsageError()
+            raise UsageError("--clicast argument is invalid, file does not exist")
+
+
+def validatePath(pathString):
+    if not os.path.isdir(pathString) and not os.path.isfile(pathString):
```

Comment:
I guess we want `or` instead of `and` here?

Reply:
I think it is ok as is because sometimes it is called with a file path and sometimes with a directory.
Let me know if I am missing something :)

Reply:
The condition might be clearer as `not (os.path.isdir(pathString) or os.path.isfile(pathString))` but it's just personal preference really, it's correct as is